samples/boards/espressif/deep_sleep/prj.conf

```diff
@@ -1 +1,2 @@
 CONFIG_POWEROFF=y
+CONFIG_GPIO=y
```

Comment:
Looks like this seem also uses GPIO optionally. The sample already has 2 Kconfig options, and only if they are enabled is GPIO used as far as I can tell. Should be trivial to add `select GPIO` to those

Reply:
fixed